bahmni-docker/.env

```diff
@@ -57,3 +57,7 @@ BAHMNI_APPS_PATH=
 
 #Bahmni Proxy Environment Variables
 PROXY_IMAGE_TAG=0.94-1
+
+#Implementer Interface Environment Variables
+IMPLEMENTER_INTERFACE_IMAGE_TAG=0.94-1
```

Comment:
`0.94-1` would give the first image not the latest... shall we move to using `latest` tag for dev

Reply:
For a developer setup `latest` should be good. But the use of specific image version tag is because when someone from community is trying out or getting started with Bahmni on Docker, it should be stable enough whereas latest in rare cases might have breaking changes.

Reply:
I agree Nouman, lets go with latest for dev, may be once we make a stable release of docker images, then we will have something like `prod.env` which would point to stable configurations.